Art/Source/3D/Sequences/PulseOx_Apply/create_pulseox_sequence.py

```
@@ -0,0 +1,562 @@
+    clear_scene()
+
+    # Setup environment
+    print("Setting up world and lighting...")
+    setup_world()
+    setup_lighting()
+    setup_camera()
+    setup_render_settings()
+
+    # Create models
+    print("Creating finger model...")
+    finger = create_finger()
```

Ruff (0.14.7)

Local variable `finger` is assigned to but never used

Remove assignment to unused variable `finger`

(F841)